docker-compose.yml

```diff
@@ -1,5 +1,11 @@
 version: '3'
 services:
+
+  memcached:
+    image: memcached:alpine
+    ports:
+      - "22122:11211"
```

Comment:
Hmm...ok, fair idea. I don't expect we would need an access here. If we you really need I presume it would be simple to just launch some bash container in the same network, have it linked and examine the memcache.